examples/swr-site/pages/remote/graphql-eslint/_meta.en-US.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-env node */
+const { listFiles } = require('../../../components/remote-utils')
+const { createCatchAllMeta } = require('nextra/catch-all')
+
+module.exports = async () => {
+  const files = await listFiles()
+  return createCatchAllMeta(
+    // Ensure you didn't forget define some file by providing all paths of remote files
+    files
+      .filter(filename => filename.startsWith('website/src/pages/docs/'))
+      .map(filename => filename.replace('website/src/pages/docs/', '')),
+    // Next you can override order of your meta files, folders should end with / and be nested
+    {
+      index: 'Introduction',
+      'getting-started': 'Getting Started',
+      'getting-started/': {
+        parser: 'Parser'
+      }
+    }
+  )
+}
```

Comment:
@shuding I still think that nested keys with slashes (I changed the order, now folders should end with slashes, like in `.gitignore`) are the most friendly way since less code is written.

With the above helper function, I was able to change the original order of items with a few lines coded

## from

<img width="293" alt="image" src="https://user-images.githubusercontent.com/7361780/213540293-d49cab6d-2c5e-4b60-8701-29383000c6b7.png">

## to
<img width="228" alt="image" src="https://user-images.githubusercontent.com/7361780/213540704-86104ee3-3ecd-493b-98e5-92fab0a853eb.png">